src/d2_docker/docker-compose.yml

```diff
@@ -13,10 +13,12 @@ services:
             - "${POST_SQL_DIR}:/data/db/post"
         environment:
             CATALINA_OPTS: "-Dcontext.path=${DEPLOY_PATH}"
-            JAVA_OPTS: "-Xmx7500m -Xms4000m"
+            JAVA_OPTS: "-Xmx7500m -Xms4000m ${JAVA_OPTS}"
             LOAD_FROM_DATA: "${LOAD_FROM_DATA}"
             DEPLOY_PATH: "${DEPLOY_PATH}"
             DHIS2_AUTH: "${DHIS2_AUTH}"
+        ports:
+            - "${CORE_DEBUG_PORT:-7000}:${CORE_DEBUG_PORT:-7000}"
```

Comment:
Right now we are exposing port 7000 always, which we shouldn't but it's not easy to conditionally forward a port